src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -103,7 +103,7 @@ class PopoverReportActionContextMenu extends React.Component {
      *
      * @param {string} type - context menu type [EMAIL, LINK, REPORT_ACTION]
      * @param {Object} [event] - A press event.
-     * @param {string} [selection] - A copy text.
+     * @param {Object} [selection] - Copied content.
```

Comment:
revert

Reply:
This can be HTML or text.

Reply:
type will be string.

Reply:
which can be HTML or text

Reply:
Ok. I got it. Let's just correct the type to string